app/celery/contact_information_tasks.py

```diff
@@ -26,41 +27,41 @@ def lookup_contact_info(
     self,
     notification_id,
 ):
-    current_app.logger.info(f'Looking up contact information for notification_id:{notification_id}.')
+    current_app.logger.info('Looking up contact information for notification_id: %s.', notification_id)
 
     notification = get_notification_by_id(notification_id)
-    va_profile_id = notification.recipient_identifiers[IdentifierType.VA_PROFILE_ID.value]
+    recipient_identifier = notification.recipient_identifiers[IdentifierType.VA_PROFILE_ID.value]
```

Comment:
why change the naming here to be less specific?

Reply:
The type of the assigned value is RecipientIdentifier (a class instance); it's not a string type.  I had to follow the downstream code to determine that and get my unit tests to pass.  I think "va_profile_id" was used here because that the name of a parameter in the downstream function in which I determined that the value is a RecipientIdentifier.  I would have changed it there too, but "va_profile_id" is used throughout that file.  I left it alone.